src/SFA.DAS.EAS.Web/Helpers/ControllerConstants.cs

```diff
@@ -6,44 +6,9 @@ public static class ControllerConstants
         public const string AccountHashedIdRouteKeyName = "HashedAccountId";
         public const string ActionKeyName = "Action";
         public const string ActivitiesUnavailableMessage = "Activities are currently unavailable.";
-        public const string AddOrganisationAddressViewName = "../OrganisationShared/AddOrganisationAddress";
```

Comment:
Where all of these rendered redundant by the removal of the "What y'all need" page? Or was there some legacy redundancy hanging around? Seems like a lot of constants removed for just one page.

Reply:
Removed as unused so tidied up code.